### DISCUSSION TITLE
Using MsgBox as modal or something similar?

### DISCUSSION THREAD

**Discussion by zagi988** [HASwitchPlate/openHASP#569] [Q&A] [answered]
Hello.
Yesterday I started with openHASP on WT32-SC01 Plus and it's going great so far.
Have the basic UI figured out, but have some problems too.

Can I somehow use MessageBox as a modal for setting the brightness lets say?
Or is there another way to make a modal (popup with some options inside) for lets say RGB controls.

The thing I want to do is that when I click a button that says "Dimming lights" that I get a popup with 2-3 sliders for setting the brightness of 2-3 dimming lights.

Can this be done somehow?

Thanks all for such a great project, hope I'll be able to contribute too.

**Comment by zagi988** [comment]
Is the modal support dropped or it is not yet implemented? Cuz I see in docs that it's crossed.

**Reply by fvanroie** [reply, marked as answer]
`msgbox` modal property has not been implemented yet. Even so, it is just for a popup message text and some buttons.
If you want to show other UI elements, take a look at tha `obj` and `parentid` properties to nest and show/hide sub UI elements.
Some examples can be found in the documentation:
https://www.openhasp.com/0.7.0/integrations/home-assistant/sampl_conf/#colored-lights-panel
or
https://www.openhasp.com/0.7.0/integrations/home-assistant/sampl_conf/#shutter-control-panel

**Reply by zagi988** [reply]
ouuuu, that's exactly what I was looking for! didn't even realize it
